_layouts/homepage.html

```diff
@@ -70,8 +70,16 @@ <h1>{{ page.secondary.heading }}</h1>
       </a>
       {% endif %}
       <a href="{{ page.ctas.roadmap.url }}" class="link-readmore" target="_blank">{{ page.ctas.roadmap.text }}</a>
-      {% assign latest = site.versions | sort: 'version_sort' | reverse | first %}
-      <div class="latest-version">{{ page.version_feature.latest_label}} {{ latest.version }} {{ page.version_feature.date_label}} {{ latest.date | date_to_string: "ordinal", "US"}}</div>
+      {% assign versions = site.versions | sort: 'version_sort' | reverse %}
+      {% assign latest = versions | first %}
+      {% for version in versions %}
+          {% assign major_version = version.version | split: "." | first %}
```

Comment:
2. Now we need to ensure the major_version is an integer. We can do that by putting a pipe and plus zero right after first.

```{% assign major_version = version.version | split: "." | first | plus: 0 %}```